src/types.d.ts

```diff
@@ -98,6 +98,7 @@ export interface FileMetaData {
   num_rows: bigint
   row_groups: RowGroup[]
   key_value_metadata?: KeyValue[]
+  geo?: GeoParquet // Partial GeoParquet metadata. `geo` will still be present in key_value_metadata
```

Comment:
This is not in the parquet spec either

Reply:
Exactly, and same comment: where could I store this? Maybe just creating a llgical type GEOMETRY when it applies?